Fotogjengen/hilfling-frontend
# Feature/taheera/actually delete archive boss stuff but not really bc soft deletes

To test: 

- Go to `localhost:3000/intern/arkivsjef`
- Open up the accordion and delete something 
- It should be deleted
